modules/t/sliceAdaptor.t

```diff
@@ -55,6 +55,13 @@ ok($slice->end   == $END);
 ok($slice->seq_region_length == 62842997);
 debug("slice seq_region length = " . $slice->seq_region_length());
 
+#
+# _create_chromosome_alias
+#
+{
+  # test that no alias can be defined for species with pre-existing chromosome coordsystem
+  throws_ok{ $slice_adaptor->_create_chromosome_alias() } qr/A chromosome CoordSystem object already exists/, 'Correctly thrown error after finding existing chromosome CoordSystem object';
+}
```

Comment:
This tests that the method throws if we already have a 'chromosome' coord_system
Can we also check that it works when there isn't one?

Reply:
Thanks yes, I will use the Parus major test db data to add more tests in when that branch is integrated.

Reply:
Test added in with commit https://github.com/Ensembl/ensembl/pull/532/commits/1c34b7d409955da23bcbae881a99d24219689964